src/styles/components/_detailsPage.scss

```diff
@@ -0,0 +1,10 @@
+$breakpoint-tablet: 820px;
+
+.cmp-specifications-list {
+  display: block;
+
+  @media only screen and (min-width: $breakpoint-tablet) {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+  }
+}
```

Comment:
Is there a `detailsPage` component? Would it be better to name this file `_specificationBlock.scss`?